index.js

```diff
@@ -18,7 +21,15 @@ const manageRecipes = async () => {
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
 
-    // Run your code here, after you have insured that the connection was made
+    await Recipe.create(bolonhesa);
+
+    await Recipe.insertMany(data)
```

Comment:
In each step they always ask you to console.log the result. A way of doing this is storing the Recipe.insertMany in a variable and then console log it:

```
    const manyRecipes = await Recipe.insertMany(data);
    manyRecipes.forEach((recipe) =>
      console.log(`New recipe added: ${recipe.title}`)
    );
```